src/page-multi-facility/types.tsx

```diff
@@ -6,3 +6,19 @@ export type Facility = {
 };
 
 export type Facilities = Facility[];
+
+export type Scenario = {
+  name: string;
+  baseline: boolean;
+  dataSharing?: boolean;
+  dailyReports?: boolean;
+  roles: Map<string, string>;
+  createdAt: TimestampType;
+  updatedAt: TimestampType;
+};
+
+type TimestampType = {
```

Comment:
I'd probably just call this `TimeStamp` unless there is a technical reason to add on the `Type` suffix.